Trappist/src/Promact.Trappist.DomainModel/ApplicationClasses/Test/TestAC.cs

```diff
@@ -45,6 +45,10 @@ public class TestAC
 
         public int NumberOfTestAttendees { get; set; }
 
+        public int NumberOfTestSections { get; set; }
+
+        public int NumberofTestQuestions { get; set; }
```

Comment:
Naming is not proper. Property declared above has a proper naming. why there is an inconsistency?